msteams-platform/apps-in-teams-meetings/build-extensible-conversation-for-meeting-chat.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Build extensible conversation for meeting chat
+author: v-sdhakshina
+description: In this article, learn how to build extensible conversation for Microsoft Teams meeting chat with bots, cards and message extensions.
+ms.topic: conceptual
+ms.author: v-sdhakshina
+ms.localizationpriority: medium
+---
+
+# Build extensible conversation for meeting chat
+
+App developers can make conversations extensible in Teams meetings. Bots, message extensions, cards and task modules can be combined to deliver an intuitive experience.
+
+## Bots
+
+A bot is also referred to as a chatbot or conversational bot. It's an app that runs simple and repetitive tasks by users such as customer service or support staff. Everyday use of bots include, bots that provide information about the weather, make dinner reservations, or provide travel information. Interactions with bots can be quick questions and answers or complex conversations. Bots that are enabled in group chat scope start functioning in meetings. To implement bots, start with [Build a bot](/microsoftteams/platform//sbs-gs-javascript?tabs=vscode%2Cvsc%2Cviscode).
```

Comment:
> Bots that are enabled in group chat scope start functioning in meetings.

Is this fully accurate? If I understood the previous doc PR correctly, channel meetings require "team" scope, right?